[msx] test Started org.apache.hadoop.hdfs.server.namenode.TestNameNodeMXBean#testNameNodeMXBeanInfo
[msx] unitTestCounterInClass = 0
2020-03-14 04:11:53,069 [main] INFO  namenode.NameNode (NameNode.java:<clinit>(931)) - [msx-restart] reconfPoint = 0
2020-03-14 04:11:53,806 [main] INFO  namenode.NameNode (NameNode.java:performReconfig(958)) - [msx-restart] NameNode init, reconfigMode is none, do nothing
2020-03-14 04:11:54,787 [main] INFO  namenode.NameNode (NameNode.java:<init>(1042)) - [msx-restart] NameNode 33011 start
2020-03-14 04:11:54,803 [main] INFO  datanode.DataNode (DataNode.java:<clinit>(433)) - [msx-restart] reconfPoint = 0
2020-03-14 04:11:54,866 [main] INFO  datanode.DataNode (DataNode.java:performReconfig(460)) - [msx-restart] DataNode init, reconfigMode is none, do nothing
2020-03-14 04:11:55,104 [main] INFO  datanode.DataNode (DataNode.java:runDatanodeDaemon(2750)) - [msx-restart] DataNode 37119 start
2020-03-14 04:11:55,117 [main] INFO  datanode.DataNode (DataNode.java:performReconfig(460)) - [msx-restart] DataNode init, reconfigMode is none, do nothing
2020-03-14 04:11:55,240 [main] INFO  datanode.DataNode (DataNode.java:runDatanodeDaemon(2750)) - [msx-restart] DataNode 37655 start
2020-03-14 04:11:56,049 [main] INFO  datanode.DataNode (DataNode.java:shutdown(2075)) - [msx-restart] DataNode 37655 stop, double check before stop, dfs.data.transfer.client.tcpnodelay = true
2020-03-14 04:11:56,092 [main] INFO  datanode.DataNode (DataNode.java:shutdown(2075)) - [msx-restart] DataNode 37119 stop, double check before stop, dfs.data.transfer.client.tcpnodelay = true
2020-03-14 04:11:56,223 [main] INFO  namenode.NameNode (NameNode.java:stop(1087)) - [msx-restart] NameNode 33011 stop, double check before stop, dfs.data.transfer.client.tcpnodelay = true
[msx] test Failure org.apache.hadoop.hdfs.server.namenode.TestNameNodeMXBean#testNameNodeMXBeanInfo
[msx] writeFile testName = org.apache.hadoop.hdfs.server.namenode.TestNameNodeMXBean#testNameNodeMXBeanInfo
[msx] failed
[msx] failureMessage: null
[msx] stackTrace: java.lang.AssertionError
2020-03-14 04:11:56,295 [main] INFO  server.JournalNode (JournalNode.java:<clinit>(168)) - [msx-restart] reconfPoint = 0
[msx] reset performed
[msx] test Finished org.apache.hadoop.hdfs.server.namenode.TestNameNodeMXBean#testNameNodeMXBeanInfo
[msx] writeFile testName = org.apache.hadoop.hdfs.server.namenode.TestNameNodeMXBean#testNameNodeMXBeanInfo
[msx] Info : file existed /root/parameter_test_controller/shared/test_results/org.apache.hadoop.hdfs.server.namenode.TestNameNodeMXBean#testNameNodeMXBeanInfo
[msx] succeed
[msx] reset performed
[msx] all testRunFinished
